### DISCUSSION TITLE
Add Data Adaptor for PrimeNG Calendar component to support Jalali calendar and others.

### DISCUSSION THREAD

**Discussion by mahmoudparandeh** [primefaces/community#559] [PrimeNG]
Hi There,
After many major releases and discussions in [https://github.com/primefaces/primeng/issues/8706](url), this feature is not implemented. Please consider adding this, so that other cultures can use this component.

**Comment by cetincakiroglu** [comment]
Hi,

Thanks for the feedback we've added it in our internal backlog.
